enhancements/installer/storage-class-encrypted.md

```diff
@@ -0,0 +1,278 @@
+---
+title: encrypted-default-storage-class
+authors:
+  - @patrickdillon
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - @jhixson74
+  - @jsafrane
+  - @bertinatto
+  - @trilokgeer
+approvers:
+  - TBD
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - TBD
+creation-date: 2022-06-22
+last-updated: 2022-06-22
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - [CORS-2049](https://issues.redhat.com/browse/CORS-2049)
+  - [OCPPLAN-9307](https://issues.redhat.com/browse/OCPPLAN-9307)
+---
+
+# Default Storage Class with Encrypted Keys 
+
+## Summary
+
+When user-managed encryption keys are provided at install time, the default storage
+class should be set to utilize those keys. Presently, only root volumes for nodes are
+encrypted with those keys--but users would expect the persistent storage volumes to
+be encrypted as well. This enhancement proposes a solution for AWS, GCP, & Azure.
```

Comment:
What is the likelihood that a user would want different encryption keys for root volumes and PVs?

Reply:
Good question, and I do not know how to determine the answer. If it is a concern, it seems possible that this proposal would still allow that by specifying the keys for PVs in the defaultMachinePool and then specifying keys for the nodes in the other machine pools.

Reply:
The only requirement we got is that both VMs and PVs should be encrypted with the same key. If users want to use different keys, they can do it as day 2 operation by creating a new StorageClass.